bin/main.rs

```diff
@@ -72,13 +72,8 @@ struct Args {
 
     /// (Advanced) Tweak the maximum number of jobs to run for handling events and commands. Be
     /// careful how you use this as it can affect performance
```

Comment:
Consider filing an issue and noting it here about setting and then exceeding the max job count causes wadm to not process more work

Reply:
https://github.com/wasmCloud/wadm/issues/141 :)